packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.tsx

```diff
@@ -247,7 +247,13 @@ const ConfirmSignupCode = ({
             goToSettingsWithAlertSuccess();
           } else {
             // Navigate to relying party
```

Comment:
Is SubPlat considered an oauth redirect now? I think it at least used to be a web integration, where we needed to check `webRedirectCheck.isValid`?